src/constants/awsBatch.ts

```diff
@@ -1,11 +1,13 @@
 //api endpoints
 const BASE_URL = "https://ng44ddk8v1.execute-api.us-west-2.amazonaws.com/testing";
 const SUBMIT_PACKING_BASE = `${BASE_URL}/submit-packing`;
+const SUBMIT_PACKING_ECS = "http://44.244.127.93:8443/pack";
 const PACKING_STATUS_BASE = `${BASE_URL}/packing-status`;
 const GET_LOGS_BASE = `${BASE_URL}/logs`;
 
-export const getSubmitPackingUrl = (recipe: string, config?: string) => {
-    let url = `${SUBMIT_PACKING_BASE}?recipe=${recipe}`;
+export const getSubmitPackingUrl = (recipe: string, config?: string, useEcs: boolean = false) => {
```

Comment:
nit: for consistency, we can pick one parameter name `useECS` or `useEcs`

Reply:
good point. I think I'll go with `useECS`, since ECS is an acronym